### ISSUE TITLE
localize application into French

### ISSUE THREAD

**Issue by hahn-kev** [sillsdev/languageforge-lexbox#507]
some feedback I got in the past is that French would be a good target language for our user base with over 200 users.

Depends on #508

**Comment by rmunn** [comment]
I speak French, and used to be at near-native fluency but these days I'd rate myself at a 4 out of 5, because I don't actually know enough vocabulary any more. (E.g., I don't know what words are in common use for computer terms like double-clicking, selecting from a menu, logging in, and so on). Still, I could help produce a first draft, and then get a native speaker to refine it.

**Comment by hahn-kev** [comment]
@rmunn would you review the translation in #516 then? Tim used machine translation to give us a starting point.

**Comment by rmunn** [comment]
The French translation looks pretty good to me. If there are any issues I missed, we can address them later in another PR, but I'd give it a 👍 .
